README.md

```diff
@@ -758,6 +758,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Dennis Quinlan](https://github.com/denquinlan)
   - [Dev Jain](https://github.com/889-dj)
   - [Devansh Tiwari](https://github.com/Devansh-365)
+  - [Devagya Rattan](https://github.com/devagya-rattan)
```

Comment:
I believe your name must be one line above (`G` comes first and later on `N`).
Let us know if you need any help with that. 🙂